llvm/tools/sycl-post-link/SYCLDeviceLibReqMask.h

```diff
@@ -34,15 +34,6 @@ enum class DeviceLibExt : std::uint32_t {
   cl_intel_devicelib_cstring,
 };
 
-class SYCLDeviceLibReqMaskPass : public ModulePass {
-public:
-  static char ID;
-  SYCLDeviceLibReqMaskPass() : ModulePass(ID) { MReqMask = 0; }
-  bool runOnModule(Module &M) override;
-  uint32_t getSYCLDeviceLibReqMask() { return MReqMask; }
-
-private:
-  uint32_t MReqMask;
-};
+uint32_t getSYCLDeviceLibReqMask(const Module &M);
```

Comment:
We should be able to remove `#include "llvm/Pass.h" as well

Reply:
Agreed. I will change it to the `#include "llvm/IR/Module.h"`, it is needed for `Module`

Reply:
Or just forward declaration.